core/src/main/java/kafka/server/builders/LogManagerBuilder.java

```diff
@@ -111,7 +111,12 @@ public LogManagerBuilder setMaxTransactionTimeoutMs(int maxTransactionTimeoutMs)
     }
 
     public LogManagerBuilder setMaxProducerIdExpirationMs(int maxProducerIdExpirationMs) {
-        this.producerStateManagerConfig = new ProducerStateManagerConfig(maxProducerIdExpirationMs);
+        this.producerStateManagerConfig = new ProducerStateManagerConfig(maxProducerIdExpirationMs, false);
```

Comment:
Should we remove `setMaxProducerIdExpirationMs` and use `setProducerStateManagerConfig` everywhere?